Closes #1078: Added UI for managing phone feature permissions

Amejia481 opened the pull request:
[Video demo](https://drive.google.com/file/d/1Vs2tWAc8my439tzrsJjELAP-pXCKKLhL/view?usp=sharing) 🍕 
### Pull Request checklist
<!-- Before submitting the PR, please address each item -->
- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

boek commented on app/src/main/res/layout/fragment_manage_site_permissions_feature_phone.xml:
```diff
@@ -0,0 +1,83 @@
+        android:layout_marginTop="@dimen/radio_button_preference_vertical">
+
+    <RadioGroup android:layout_width="match_parent"
+                android:layout_height="wrap_content">
```
One nit: Tab size on some of these files are 8 spaces instead of 4

Amejia481 replied:
Thanks I will fix it!

Amejia481 replied:
Fixed

Amejia481 commented:
I fixed the indentation issue and added an extra commit   [Closes #1078: Added UI for managing phone feature permissions](https://github.com/mozilla-mobile/fenix/issues/1229) for fixing a related issue.

boek commented:
Thanks @Amejia481!
